### ISSUE TITLE
Theme Dev: Add Style for Edit fields menu on Game Menu

### ISSUE THREAD

**Issue by TheKersalMassive** [JosefNemec/Playnite#3123]
### Feature description

Currently the edit fields in the Game Menu > Edit menu don't have a style that is customisable in GameMenu.xaml, and this means they display the current default button style.

For the default theme this is fine as the default button looks the same as the custom themes, if the default button is specifally designed to be different to the rest of the UI, i.e.

**Comment by JosefNemec** [comment]
There already is a style for those `ItemSelectionButton` in `ItemSelection.xaml`.

**Comment by TheKersalMassive** [comment]
Yep, you're right, seems I've missed those from my theme update to P10:

A source/Playnite.FullscreenApp/Themes/Fullscreen/Default/Views/HelpMenu.xaml
A source/Playnite.FullscreenApp/Themes/Fullscreen/Default/Views/ItemSelection.xaml
